tests/test_async_client.py

```diff
@@ -0,0 +1,180 @@
+import unittest
+import logging
+import asyncio
+from unittest.mock import AsyncMock, patch
+import unittest.async_case
+import willofsteel
+from willofsteel.async_client import AsyncClient
+from willofsteel.exceptions import *
+from willofsteel.types import *
+
+
+class TestAsyncClient(unittest.async_case.IsolatedAsyncioTestCase):
+
+    def setUp(self):
+        patcher = patch(
+            'willofsteel.async_client.AsyncClient.request', new_callable=AsyncMock)
+        self.addCleanup(patcher.stop)
+        self.mock_request = patcher.start()
+        self.mock_request.return_value = {'success': True,
+                                          'user_id': 707274479751135243}, 200
+        self.client = AsyncClient(
+            'valid_key', logger=LoggingObject(level=logging.CRITICAL))
+
+    def tearDown(self):
+        asyncio.run(self.client.close())
+        super().tearDown()
+
+    @ patch('willofsteel.async_client.AsyncClient.request', new_callable=AsyncMock)
```

Comment:
```suggestion
    @patch('willofsteel.async_client.AsyncClient.request', new_callable=AsyncMock)
```

Please don't add a space between @ and the patch decorator